developer-docs-site/docs/tools/aptos-cli/use-cli/use-aptos-cli.md

```diff
@@ -97,59 +97,67 @@ Help for sub-commands is also available. For example, see below the usage output
 
 ```bash
 
-USAGE:
-    aptos move compile [OPTIONS]
-
-OPTIONS:
-        --bytecode-version <BYTECODE_VERSION>
```

Comment:
Let's follow up and revamp the areas below about how to use it for common tasks perhaps?

Reply:
Yeah, I have been thinking of how to organize this page and the one from move language page. Maybe we can link to the Aptos CLI usage under move language - https://aptos.dev/move/move-on-aptos/cli/